tests/deprecated_api/test_remove_1-3.py

```diff
@@ -135,3 +135,7 @@ def test_trainer_cli_profiler_remove_in_v1_3_0(cli_args, expected_parsed_arg, ex
     assert getattr(args, "profiler") == expected_parsed_arg
     trainer = Trainer.from_argparse_args(args)
     assert isinstance(trainer.profiler, expected_profiler)
+
+
+def test_trainer_enable_pl_optimizer(tmpdir):
+    Trainer(enable_pl_optimizer=True)
```

Comment:
this doew not test anything :]
```suggestion
    with pytest.deprecated_call(match='will be removed in v1.3'):
        Trainer(enable_pl_optimizer=True)
```